cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java

```diff
@@ -0,0 +1,42 @@
+package com.commonsware.cwac.cam2.helper;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.util.SparseArray;
+
+import com.google.android.gms.vision.Frame;
+import com.google.android.gms.vision.face.Face;
+import com.google.android.gms.vision.face.FaceDetector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ImageHelper {
+  public boolean isFaceOccupancy(Context context, Bitmap bitmap, float percent) {
```

Comment:
return an enum for following cases
1. no face found
2. more than face found
3. one face found but smaller than the required percentage
4. Face found with required percentage